packages/components/pie-card-container/src/index.ts

```diff
@@ -58,12 +63,46 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 aria-label=${this.aria?.label || nothing}
-                aria-disabled=${this.disabled ? 'true' : 'false'}>
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                style=${paddingCSS || nothing}>
                     <slot></slot>
                 </div>
             </a>`;
     }
 
+    /**
+     * Generates padding for the component based on `padding` values passed
+     * by the consumer.
+     *
+     * Example: 'a' or 'a, b'
```

Comment:
Same as in the docs, I would just put here that single applies it to all sides and `a, b` as `leftRight, topBottom`